### PR TITLE
fix idam redirect url

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###
fix idam redirect url

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ci-values.yaml [hunk 1]
```diff
@@ -5,9 +5,10 @@ global:
 
 rpe-service-auth-provider:
   java:
-    MICROSERVICEKEYS_CCD_ADMIN: AAAAAAAAAAAAAAAA
-    MICROSERVICEKEYS_CCD_DATA: AAAAAAAAAAAAAAAA
-    MICROSERVICEKEYS_CCD_DEFINITION: AAAAAAAAAAAAAAAA
+    environment:
+      MICROSERVICEKEYS_CCD_ADMIN: AAAAAAAAAAAAAAAA
+      MICROSERVICEKEYS_CCD_DATA: AAAAAAAAAAAAAAAA
+      MICROSERVICEKEYS_CCD_DEFINITION: AAAAAAAAAAAAAAAA
```
Comment (nitinprabhu) [inline ci-values.yaml:11]: Is this change required ?

**Review by nitinprabhu — APPROVED** [review]

File: ci-values.yaml (hunk 1 above)
Comment (mdayican) [reply ci-values.yaml:11]: syntax was wrong . but it seems we do not need  these values for ci.
